test/scenarios.spec.ts

```diff
@@ -221,3 +233,94 @@ suite(
         });
     },
 );
+
+suite('== Custom Handler Function scenarios ==', () => {
+    class DateSelectorManager extends ControlManager {
+        createControlTree(state: any): Control {
+            const topControl = new ContainerControl({ id: 'root' });
+
+            // DateControl
+            const dateControl = new DateControl({
+                id: 'dateControl',
+                interactionModel: {
+                    targets: [$.Target.Date],
+                    actions: {
+                        set: [$.Action.Set],
+                        change: [$.Action.Change],
+                    },
+                },
+                inputHandling: {
+                    customHandlingFuncs: [
```

Comment:
how do you feel about:
```
customHandlingFuncs: [
    {canHandle: isSetDateEvent, handle: handleSetDateEvent},
    {canHandle: isSetValue, handle: handleSetValue},

],
```

Reply:
But having inputHandling has its own advantages per our discussions:
- There could be more things that fall under input handling in future (confidence threshold, input confirmation strategy, domain SPIs, multi-sensory (presence detection and more), skill connections, etc.

Reply:
Looks more cleaner and helps in IntelliSense, updated